### DISCUSSION TITLE
Clickable product images bigger (kind of popup) on the admin

### DISCUSSION THREAD

**Discussion by lilianamc** [PrestaShop/PrestaShop#40150] [Ideas]
It would be nice to have clickable product images bigger (kind of popup) on the admin. Some products have small differences and the product image on the order (admin) is very small. So I have to go to the products page to see what is the product.  Please see image. https://imgur.com/a/DAnIHW0

Thank you very much

**Comment by khouloudbelguith** [comment]
Hi @lilianamc,

This feature could be added by a custom module.
@marion what do think?
Thanks!

**Comment by MatShir** [comment]
Hi @lilianamc,

It is an interesting feature. Just to understand your need, you don't rely on the ref of your product or his name when you do order management?

We got your feedback and we have shared the idea at Prestashop Feedbacks (http://feedback.prestashop.com/forums/387864-prestashop-1-7-x/suggestions/36890221-zoom-on-product-images-on-the-order-list/) to let the community help us to prioritizes all ideas from the community.

Thank you,

**Comment by lilianamc** [comment]
Hi @MatShir , 

Yes, you are right. It 's much easier to click on the image and see it bigger to avoid mistakes.

**Comment by lilianamc** [comment]
Hi!

Don't forget this nice and helpful feature :)
